src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md

```diff
@@ -0,0 +1,92 @@
+# Migration Guide - Standalone Application to DX Experience Cloud SPA v1
+
+## Overview 
+
+In this guide, you will learn how to migrate an App Builder application that was originally initialized as a Standalone Application to a 
+DX Experience Cloud SPA v1. 
+
+## Who is This Migration Guide For?
+
+If you are having trouble viewing your application in Adobe Experience Cloud, you may have initialized your application as a 
+Standalone Application, and this guide may help you. 
+
+Applications built with App Builder can only be accessed via the App Builder Catalog in Adobe Experience Cloud if they have been initialized as **DX 
+Experience Cloud SPA v1**. 
+
+## How can I tell if my project is a Standalone Application?
+
+Your application will **not** have the `App Builder ExcShell` sub-heading in Adobe Exchange. 
+
+### Standalone Application 
+![image](https://user-images.githubusercontent.com/28722775/173436533-7adf7cf4-c783-41eb-8e80-d78a91c13227.png)
```

Comment:
I don't think you should use the Magento logo here.

Reply:
Screenshot has been switched out, let me know if this one works.